### PR TITLE
Add Circuit to list of multiplatform architectures

### PR DESCRIPTION
**Pull request by ZacSweers**
One of the Circuit authors 👋. Would love to list our project here if you're open to it/looking for more examples.

### CODE REVIEW

**Review by SebastianAigner — APPROVED** [review]

File: topics/compose/compose-navigation-routing.md [hunk 1]
```diff
@@ -11,5 +11,6 @@ you can choose from:
 | [Decompose](https://arkivanov.github.io/Decompose/) | An advanced approach to navigation that covers the full lifecycle and any potential dependency injection |
 | [Appyx](https://bumble-tech.github.io/appyx/)       | Model-driven navigation with gesture control                                                             |
 | [PreCompose](https://tlaster.github.io/PreCompose/) | A navigation and view model inspired by Jetpack Lifecycle, ViewModel, LiveData, and Navigation           |
+| [Circuit](https://slackhq.github.io/circuit/)       | A Compose-driven architecture for Kotlin and Android applications.                                       |
```
Comment (zamulla) [inline topics/compose/compose-navigation-routing.md:14]: On this page, we're mentioning tools that help with navigation and routing. Could you say a couple of words about the way Circuit helps in this regard specifically?

Comment (ZacSweers) [reply topics/compose/compose-navigation-routing.md:14]: Sure, in this PR or just in this comment for your awareness?

Comment (zamulla) [reply topics/compose/compose-navigation-routing.md:14]: Sorry, I missed your reply! In the PR itself, yes, if you could.

Comment (ZacSweers) [reply topics/compose/compose-navigation-routing.md]: Done! Sorry for teh delay, I'd completely forgotten this

**Review by zamulla — APPROVED** [review]

**Comment by ZacSweers** [comment]
Thanks!

**Pull request merged**